mobile/src/services/volunteer/volunteer.service.ts

```diff
@@ -47,12 +47,14 @@ export const useJoinByAccessCodeMutation = () => {
             activeOrganization: {
               ...data.organization,
               volunteerId: data.id,
+              volunteerProfileId: null,
```

Comment:
Nu exista posibilitatea sa avem volunteerProfileId pe `data: IVolunteer`?

Reply:
as spune ca nu prea, in momentul in care abia dam join unei organizatii, nu avem un profil de voluntar, il facem abia dupa.

Reply:
Avem un caz de `rejoin` in momentul in care un voluntar da `Leave organization` si apoi da din nou `Join` noi ii pastram volunteerId. Ar putea sa intre in cazul asta?

Reply:
when rejoining an organization, we no longer interfere with the useJoinByAccessCodeMutation, therefore I believe that the volunteerProfileId can only be null at this point